### ISSUE TITLE
Is there no way to create custom events via this SDK?

### ISSUE THREAD

**Issue by mikesnare** [HubSpot/hubspot-api-nodejs#490]
I see that you can _send_ custom events using this SDK, but I don't see any way to create custom event definitions.  Am I just missing it?

**Comment by Sakib25800** [comment]
You're not missing it, there's no available methods to create it. You'll have to resort to `hs.apiRequest({...})`. I think since custom events are in beta they're not going to add the methods to the SDK just yet. 

If you head over to the [docs](https://developers.hubspot.com/beta-docs/reference/api), it'll show you whether the request can be done via the SDK or not.

**Comment by ksvirkou-hubspot** [comment]
Hi @mikesnare 
Do you mean [Send custom event completion](https://developers.hubspot.com/beta-docs/reference/api/analytics-and-events/custom-events/custom-event-completions#post-%2Fevents%2Fv3%2Fsend)?
`hubspotClient.events.send.customEventDataApi.send()`

**Comment by mikesnare** [comment]
@ksvirkou-hubspot Nope.  I mean management APIs for the event definitions themselves.  I was able to switch to apiRequest as suggested for now.  Thanks!
